ql/src/java/org/apache/hadoop/hive/ql/io/parquet/read/DataWritableReadSupport.java

```diff
@@ -278,15 +278,7 @@ public static Boolean getWriterDateProleptic(Map<String, String> metadata) {
       return null;
     }
     String value = metadata.get(DataWritableWriteSupport.WRITER_DATE_PROLEPTIC);
```

Comment:
I don't really understand the code fully but it looks like
 `String value = metadata.get(DataWritableWriteSupport.WRITER_DATE_PROLEPTIC);` should have been inside the try catch block right?

Reply:
Thanks for taking a look @achennagiri . 

I don't think we need a try catch for `metadata.get`. It's just a `Map` of `String` objects so there is no way to throw an exception especially `DateTimeException`. It seems to be just a bad copy paste from the method above (`getWriterTimeZoneId`).

Reply:
Correct! Thanks for the explanataion!